packages/strapi-design-system/src/ToggleInput/ToggleInput.tsx

```diff
@@ -50,19 +182,27 @@ export const ToggleInput = ({
     <FieldWrapper name={name} hint={hint} error={error} id={generatedId} required={required}>
       <Flex direction="column" alignItems="stretch" gap={1}>
         <Flex>
-          <FieldLabel action={labelAction}>{label}</FieldLabel>
+          {label ? <FieldLabel action={labelAction}>{label}</FieldLabel> : null}
           {clearLabel && onClear && checked !== null && !disabled && (
             <ClearButton onClick={onClear}>{clearLabel}</ClearButton>
           )}
         </Flex>
-        <ToggleCheckbox id={generatedId} size={size} checked={checked} disabled={disabled} {...props}>
-          {label}
-        </ToggleCheckbox>
+        <ToggleInputInput checked={checked} disabled={disabled} {...props} />
         <FieldHint />
         <FieldError />
       </Flex>
     </FieldWrapper>
   );
 };
 
-ToggleInput.displayName = 'ToggleInput';
+const FieldWrapper = styled(Field)`
+  max-width: 320px;
+`;
```

Comment:
Since `Field` is of type `Box` now I think you can pass maxWidth as a prop instead.